### DISCUSSION TITLE
Tailwind UI with NativeWind for crossplatform UI components?

### DISCUSSION THREAD

**Discussion by redbar0n** [nativewind/nativewind#275] [Q&A] [answered]
What's the chance that [Tailwind UI](https://tailwindui.com/components/preview) could work on native by using NativeWind?
Could something like "NativeWind UI" be a clean rewrite of Tailwind UI components, but with `<View>`'s instead of `<div>`'s etc.? What would be the etcetera?

**Comment by marklawlor** [comment, marked as answer]
The Tailwind UI styles should largely be copy/pastable into your NativeWind project, so it's a great start to building your own components.

However `@headlessui/react` (the JS library that powers many Tailwind UI components) is largely web based. Glancing at the docs about 6/10 components use React-Dom only features like React portals/animations.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
